benchmark/README.md

```diff
@@ -31,8 +33,145 @@ npm run benchmark -- -u 4 -e 15 -r 5
+pre-trained model, you can simply load and return said model in the function via `tf.loadLayersModel(modelPath)`.
+
+```js
+import * as tf from '@tensorflow/tfjs'
+
+import { Task } from '../task'
+
+export const task: Task = {
+  taskID: 'simple_face',
+  displayInformation: {
+    taskTitle: 'Simple Face',
+    summary: 'Can you detect if the person in a picture is a child or an adult?',
+    ...
+  },
+  trainingInformation: {
+    modelID: 'simple_face-model',
+    epochs: 50,
+    roundDuration: 1,
+    validationSplit: 0.2,
+    batchSize: 10,
+    preprocessFunctions: [],
+    learningRate: 0.001,
+    modelCompileData: {
+      optimizer: 'sgd',
+      loss: 'categoricalCrossentropy',
+      metrics: ['accuracy']
+    },
+    dataType: 'image',
+    csvLabels: false,
+    IMAGE_H: 200,
+    IMAGE_W: 200,
+    LABEL_LIST: ['child', 'adult'],
+    scheme: 'Federated'
+  }
+}
+
+export function model (): tf.LayersModel {
+  // Init model
+  const model = tf.sequential()
+
+  // Add layers
+  model.add(...)
+
+  return model
+```
+
+After adding the `simple_face.ts` task class, we also need to export it in the `index.ts` file which lives in the same folder.
+
+The contents of `index.ts` are as follows
+
+```js
+export * as cifar10 from './cifar10'
+export * as lus_covid from './lus_covid'
+export * as mnist from './mnist'
+export * as simple_face from './simple_face' // <---- 
+export * as titanic from './titanic'
+```
+
+This allows us to access our task as via discojs:
 
-Currently only the simple-face task is supported but different tasks will be available through the CLI 
-in the future.
+```js
 
+import { tasks } from 'discojs'
+
+const TASK = tasks.simple_face.task
+```
+
+> Note that you need to rebuild discojs every time you make changes to it (`cd discojs; rm -rf dist/; npm run build`).
+
+### Dataset
+
+The only thing missing is loading the data, we use our own data class that is a wrapper for the `tfjs` dataset class.
+
+The key class (for images) is the ImageLoader, this needs to be extended since files are loaded differently depending
+on the environment (node vs browser). Note this is also where we can add pre-processing, here we simply normalise the 
+images, but more complex operations can also be added.
+
+Once we have built this object we can load the dataset by giving as an input the `files: string[]` and `labels: number[]`.
+We give an example of what this looks like down bellow.
+
+```js
+import * as tf from '@tensorflow/tfjs-node'
+import fs from 'fs'
+import {dataset, Task} from 'discojs'
+
+class NodeImageLoader extends dataset.ImageLoader<string> {
+  async readImageFrom(source: string): Promise<tf.Tensor3D> {
+    const imageBuffer = fs.readFileSync(source)
+    let tensor = tf.node.decodeImage(imageBuffer)
+    // <---- Add pre processing here!
+    // e.g: If resize needed uncomment the following
+    // tensor = tf.image.resizeBilinear(tensor, [
+    //   32, 32
+    // ])
+    tensor = tensor.div(tf.scalar(255))
```

Comment:
can't this be shared with the browser data-preprocessing somehow - or at least people should understand these two preprocessings should/must be the same. otherwise we can't claim it's the same task

Reply:
yeah it could be, it's something we wanted to add with @s314cy, but I think we forgot about it since we didn't add it as an issue. @s314cy correct me if im wrong.
And indeed it would be not be identical with the browser.